src/main/java/net/neoforged/neoforge/capabilities/RegisterCapabilitiesEvent.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.capabilities;
+
+import java.util.ArrayList;
+import java.util.Objects;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.EntityType;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.ItemLike;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.entity.BlockEntityType;
+import net.neoforged.bus.api.Event;
+import net.neoforged.fml.event.IModBusEvent;
+
+/**
+ * Fired to register capability providers at an appropriate time.
+ */
+public class RegisterCapabilitiesEvent extends Event implements IModBusEvent {
+    RegisterCapabilitiesEvent() {}
+
+    // BLOCKS
+
+    /**
+     * Register a capability provider for some blocks.
+     * <p>
```

Comment:
Prefix the paragraphs with `<p>`, rather than those tags being on their lines.

(Same comment for the other javadoc comments.)